.ci/cico_build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright (c) 2017 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+# Just a script to get and build eclipse-che locally
+# please send PRs to github.com/kbsingh/build-run-che
```

Comment:
Douh.... Seems we have a lot of copy-paste which should be revised...
But it's OK to merge some working state and create a separate issue to clean scripts up